features/desktop/likes.feature

```diff
@@ -0,0 +1,37 @@
+@javascript
+Feature: Liking posts
+  Background:
+    Given following users exist:
+      | username    | email             |
+      | Bob Jones   | bob@bob.bob       |
+      | Alice Smith | alice@alice.alice |
+    And a user with email "bob@bob.bob" is connected with "alice@alice.alice"
+    And "bob@bob.bob" has a public post with text "I like unicorns"
+    And I sign in as "alice@alice.alice"
+
+  Scenario: Liking and unliking a post from the stream
+    When I am on the home page
+    Then I should see "Bob"
+    And I should see "I like unicorns"
+    When I like the post "I like unicorns" in the stream
+    Then I should see "1 Like" within "#main_stream .stream_element .likes"
+
+	When I unlike the post "I like unicorns" in the stream
+    Then I should not see "1 Like" within "#main_stream .stream_element .likes"
```

Comment:
That is the only failing line because after unliking the post cucumber isn't able to find `.likes` anymore so the correct step would be
`Then I should not see a ".likes" within "#main_stream .stream_element"`